packages/core/src/awsService/appBuilder/serverlessLand/metadataManager.ts

```diff
@@ -62,9 +64,18 @@ export class MetadataManager {
                 const metadataContent = nodefs.readFileSync(metadataPath, { encoding: 'utf-8' })
                 const parseMetadata = JSON.parse(metadataContent) as ProjectMetadata
                 this.metadata = parseMetadata
+                telemetry.record({
+                    action: 'serverlessland_metadataLoaded',
+                    result: 'Succeeded',
+                })
             }
             return this.metadata
         } catch (err) {
+            telemetry.record({
```

Comment:
Since this is all wrapped in `telemetry.lambda_createServerlessLandProject.run`, you don't need to do any of this. `result`, `reason` will be populated automatically.

All you need to do is record `action` at a higher level, and then it will be a part of the Success and Failure metric